### ISSUE TITLE
BOOST NUKE: Failing to perform `boost nuke` when event code has changed

### ISSUE THREAD

**Issue by LaiaPerez88** [boostercloud/booster#62]
**Description**
**Given** that I am a user with a Booster project deployed
**When** I modify one of my events
**Then** I should still be able to nuke my application
**Current behavior:**

When I change the directory of my event
And I try to nuke my application
Then an error is returned
`✖ Error registering reducer: The event CartChanged was already registered to be reduced by method projectCartChanged in the entity Cart.
`
**Solution to problem**

1. Remove `dist` folder
2. execute `boost nuke`

**Expected behavior:**

The application can be deleted without having to remove `dist` folder.

**Comment by glammers1** [comment]
@NickSeagull @tainguyenbui Hi, it seems this issue was already addressed by Nick here #56. As he said (it happens for both deploys and nukes):

> I thought about forcing a clean build on deploy, but that would be too slow for many cases. Added a better hint to the error message.

This issue was opened based on an older version than Nick's changes. Now, messages are as follows:

```
✖ Error: A command called ChangeCart is already registered.
        If you think that this is an error, try performing a clean build.
```

So, if you execute `npm run clean` as the error message points out, you will be able to nuke your application.

In the case of a `nuke`, we also have to read the project config because it's needed to execute the `assemble` in order to get the stack service configuration, so I think we can't skip that step, which is the one that throws the error.

What do you think? Am I right?

**Comment by NickSeagull** [comment]
Yep, thanks!
